src/encoded/static/components/award.js

```diff
@@ -1926,14 +1930,14 @@ class CumulativeGraph extends React.Component {
                 data: {
                     labels: monthReleased,
                     datasets: [{
-                        label: 'Date Submitted',
-                        data: submitteddatavalue,
-                        backgroundColor: 'rgba(54, 162, 235, 0.2)',
-                    },
-                    {
                         label: 'Date Released',
                         data: releaseddatavalue,
-                        backgroundColor: 'rgba(75, 192, 192, 0.2)',
+                        backgroundColor: '#604a7b',
```

Comment:
shouldn't these colors be mapped in a module somewhere, rather than hard coded.   This is just a future refactoring suggestion.